core/src/main/resources/db/cgds.sql

```diff
@@ -177,16 +178,34 @@ CREATE TABLE `sample_list_list` (
 );
 
 -- --------------------------------------------------------
+CREATE TABLE `genetic_entity` (
+  `INTERNAL_ID` int(11) NOT NULL AUTO_INCREMENT,
+  `ENTITY_TYPE` varchar(45) NOT NULL,
+  PRIMARY KEY (`INTERNAL_ID`)
+);
+
+-- --------------------------------------------------------
 CREATE TABLE `gene` (
+  `GENETIC_ENTITY_ID` int(11) NOT NULL,
   `ENTREZ_GENE_ID` int(11) NOT NULL,
   `HUGO_GENE_SYMBOL` varchar(255) NOT NULL,
   `TYPE` varchar(50),
   `CYTOBAND` varchar(50),
   `LENGTH` int(11),
   PRIMARY KEY (`ENTREZ_GENE_ID`),
-  KEY `HUGO_GENE_SYMBOL` (`HUGO_GENE_SYMBOL`)
+  UNIQUE KEY `GENETIC_ENTITY_ID_UNIQUE` (`GENETIC_ENTITY_ID`),  
+  KEY `HUGO_GENE_SYMBOL` (`HUGO_GENE_SYMBOL`),
```

Comment:
note: here I am *not* changing anything in the KEY `HUGO_GENE_SYMBOL`, it just moved down one line.